packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.spec.ts

```diff
@@ -131,9 +150,7 @@ describe('NodeHttpRequestResponseHandler', () => {
       await handler.handle(streamMock).toPromise();
       expect(nestedHttpHandler.handle).toHaveBeenCalledTimes(1);
 
-      expect(nestedHttpHandler.handle).toHaveBeenCalledWith(expect.objectContaining({
-        request: { url: new URL('http://localhost:3000/test?works=yes'), method: 'GET', headers: {} },
-      }));
+      expect(nestedHttpHandler.handle).toHaveBeenCalledWith({ request: { url: new URL('http://localhost:3000/test?works=yes'), method: 'GET', headers: {}, body: 'bodyString' } });
```

Comment:
You left out `objectContaining` here, while in fact it is very usefull. A lot of the other tests could be shorter and more precise by using this Jest function to only specify what is expected by that test, instead of also repeating all 'irrelevant' mock data every time.

Reply:
The reason I left it out was simply because it didn't work for me. 
`expect(nestedHttpHandler.handle).toHaveBeenCalledWith(expect.objectContaining({ request: { url: new URL('http://localhost:3000/test?works=yes'), method: 'GET', headers: {} } }));`
results in:
![image](https://user-images.githubusercontent.com/43341653/133445320-575c6a72-bf00-457a-848b-ecc58f048850.png)
Am I using it wrong? It seems like the same way jonas used it to me?

Reply:
Since the object containing the body is not the top-level object, but one nested in `request`, you will also have to nest the `expect` calls, like in [the docs](https://jestjs.io/docs/expect#expectobjectcontainingobject).